CMakeLists.txt

```diff
@@ -371,7 +364,7 @@ target_link_libraries(libopenmc ${ldflags} ${HDF5_LIBRARIES} ${HDF5_HL_LIBRARIES
 
 if(dagmc)
   target_compile_definitions(libopenmc PRIVATE DAGMC)
-  target_link_libraries(libopenmc dagmc-imported)
+  target_link_libraries(libopenmc dagmc-shared pyne_dagmc-shared uwuw-shared)
```

Comment:
Why does `pyne_dagmc-shared` need to be included?

Reply:
It seems it isn't. I'd mistakenly thought it was necessary due to the references to all of the PyNE objects in the UWUW library. Thanks for catching that!